src/app/views/sidebar/history/History.tsx

```diff
@@ -252,6 +276,15 @@ export class History extends Component<IHistoryProps, any> {
     };
   }
 
+  private deleteCategoryHistory = (category: string) => {
+    const { groupedList } = this.state;
+    const queries = groupedList.items;
```

Comment:
You can use a one liner here

`const { groupedList: { items } } = this.state`

Reply:
It is not as easily readable